build_config/webpack.dev.conf.js

```diff
@@ -1,43 +1,50 @@
-var ExtractTextPlugin = require("extract-text-webpack-plugin");
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
 module.exports = {
-  devtool: "source-map",
+  entry: './public/video-ui/src/app.js',
+  mode: 'development',
+  devtool: 'source-map',
   module: {
-    loaders: [
+    rules: [
       {
         test: /\.js$/,
         exclude: /node_modules/,
-        loader: "babel-loader?cacheDirectory=true"
+        loader: 'babel-loader?cacheDirectory=true'
       },
       {
-        test: /\.scss$/,
-        use: ["style-loader", "css-loader?sourceMap!sass-loader?sourceMap"]
-      },
-      {
-        test: /\.css$/,
-        use: ["style-loader", "css-loader?sourceMap"]
+        test: /\.(sa|sc|c)ss$/,
+        use: [
+          {
+            loader: MiniCssExtractPlugin.loader
+          },
+          'css-loader',
+          'sass-loader'
+        ]
```

Comment:
This compiles both css and scss: https://webpack.js.org/plugins/mini-css-extract-plugin/